src/coreclr/jit/emitarm64.cpp

```diff
@@ -24628,7 +24935,7 @@ BYTE* emitter::emitOutput_InstrSve(BYTE* dst, instrDesc* id)
             code = emitInsCodeSve(ins, fmt);
             code |= insEncodeReg_V_4_to_0(id->idReg1());                                           // ddddd
             code |= insEncodeReg_V_9_to_5(id->idReg2());                                           // nnnnn
-            code |= insEncodeSveElemsize_tszh_22_tszl_20_to_19(optGetSveElemsize(id->idInsOpt())); // xx
+            code |= insEncodeSveElemsize_tszh_23_tszl_20_to_19(optGetSveElemsize(id->idInsOpt())); // xx
```

Comment:
The existing uses for `insEncodeSveElemsize_tszh_22_tszl_20_to_19` only assumed 1/2/4 byte sizes and not 8, so we should probably put an assert, before it `assert(optGetSveElemsize(id->idInsOpt()) == EA_8BYTE)`, **or** we could just have two functions for `tszh:tszl` with one that allows 1/2/4 byte and the other 1/2/4/8 byte.

Reply:
I prefer the assert approach; I'll add those in.

Reply:
+1 for asserts. The size restriction is (usually) a property of the instruction operation, whereas the helper function only cares about bit encodings.